api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -232,6 +232,15 @@ const (
 
 	// JSONPatchAnnotation allow modifying the kubevirt VM template using jsonpatch
 	JSONPatchAnnotation = "hypershift.openshift.io/kubevirt-vm-jsonpatch"
+
+	// KubeAPIServerGOGCAnnotation allows modifying the kube-apiserver GOGC environment variable to impact how often
```

Comment:
these variables docs for the record https://pkg.go.dev/runtime

Reply:
Thx @enxebre  I will put that link in the doc in a follow up PR